packages/wrangler/src/cloudchamber/deploy.ts

```diff
@@ -0,0 +1,168 @@
+import { readFileSync, statSync } from "fs";
+import path from "path";
+import { type Config } from "../config";
+import { type ContainerApp } from "../config/environment";
+import { UserError } from "../errors";
+import { CI } from "../is-ci";
+import { type Logger } from "../logger";
+import { fetchVersion } from "../versions/api";
+import { apply } from "./apply";
+import { build } from "./build";
+import { fillOpenAPIConfiguration } from "./common";
+
+export async function buildContainers(config: Config, workerTag: string) {
```

Comment:
What is `workerTag` here? It seems like this is being passed `versionId`, `workerTag`, or `"worker-tag"` depending on context

Reply:
this is the image tag, renamed